### ISSUE TITLE
[BUG] Scoreboard problems

### ISSUE THREAD

**Issue by K4ryuu** [KitsuneLab-Development/K4-System-MMRanks#9]

**Comment by inFqm** [comment]
Hello K4ryuu. Did have enough time to look into?

**Comment by K4ryuu** [comment]
Yes, I did have time long time ago and also we discussed it further on my Discord server. You find the discussion here: https://discord.com/channels/806961304979898368/1176940452956618853/1241682893449723945
